src/FormElement/RadioOption.php

```
@@ -0,0 +1,148 @@
+<?php
+
+namespace ipl\Html\FormElement;
+
+use ipl\Html\Attribute;
+use ipl\Html\Attributes;
+
+class RadioOption
+{
+    public const LABEL_CLASS = 'radio-label';
+
+    /** @var string|int|null Value of the option */
+    protected $value;
+
+    /** @var string Label of the option */
+    protected $label;
+
+    /** @var mixed Css class of the option's label */
+    protected $labelCssClass;
+
+    /** @var bool Whether the radio option is disabled */
+    protected $disabled = false;
+
+    /** @var Attributes */
+    protected $attributes;
+
+    /**
+     * RadioOption constructor.
+     *
+     * @param string|int|null $value
+     * @param string $label
+     */
+    public function __construct($value, string $label)
+    {
+        $this->value = $value === '' ? null : $value;
+        $this->label = $label;
+    }
+
+    /**
+     * Set the label of the option
+     *
+     * @param string $label
+     *
+     * @return $this
+     */
+    public function setLabel(string $label): self
+    {
+        $this->label = $label;
+
+        return $this;
+    }
+
+    /**
+     * Get the label of the option
+     *
+     * @return string
+     */
+    public function getLabel(): string
+    {
+        return $this->label;
+    }
+
+    /**
+     * Get the value of the option
+     *
+     * @return int|string|null
+     */
+    public function getValue()
+    {
+        return $this->value;
+    }
+
+    /**
+     * Add css class to the option label
+     *
+     * @param string|string[] $labelCssClass
+     *
+     *  @return $this
+     */
+    public function addLabelCssClass($labelCssClass): self
```

Rename this to `setLabelCssClass` please. The passed value should replace the default.

Anyone who still wants the default class, can do this: `$option->setLabelCssClass([RadioOption::LABEL_CLASS, 'new'])`

We don't provide a default style for `radio-label`, so there's nothing one could want to *keep*. And my suggestion recently for Yonas' problem would have the default made worse, because he would need to revert most of the CSS rules defined for the default class in order to hide the option. Even worse it could lead to this: `.radio-label:not(.hidden)`

So, being able to remove the default should be possible. Keeping the default also, which is the case by having the constant public.